Text/Wording Improvements (#11)
1010todd/Mega-Freight: data/mmissions.txt

	invisible
	on offer
		set "global: Mega Freight Installed"

> 		accept

Author: Don't they automatically decline?

> 		accept

Author: ```suggestion
```

	landscape land/space0
	attributes station
	description `To further advance the Republic Navy and secure humanity's future, the government is building a new orbital naval yard to allow production of larger and more advanced ships and remains easily upgradable for any future improvement and expansion.`

> 	spaceport `The spaceport have few basic accommodations, mostly as resting areas for construction workers.`

Author: ```suggestion
	spaceport `The spaceport has a few basic facilities, mostly as resting area for construction workers.`
```

	security .8

> # Random note: based on calculations a complete ring would have 3556 docks which means up to 3556 superheavies can be built concurrently at a time.

Author: ```suggestion
#Random note: based on calculations a complete ring would have 3556 docks which means up to 3556 superheavies can be built concurrently at a time.
```

> 

Author: ```suggestion
```

	planet "Canaveral Naval Yard"
		landscape land/space0
		attributes station
		description `To further advance the Republic Navy and secure humanity's future, the government built a new orbital naval yard to allow production of larger and more advanced ships. It is also designed to be easily upgradable for any future improvement and expansion. The naval yard is very heavily armed with countless numbers of artillery turrets and advanced anti-missile systems spread across the surface. There's also always a fleet carrier in nearby orbit, always within clear visual range.`

> 		spaceport `The spaceport has a few basic accommodations, mostly resting areas for the shipyard workers.`

Author: It makes it so that they will make you pay to try to land, a.k.a. you have to pay almost twice your net worth.
No landing on this station if the Republic is hostile.

> 		spaceport `The spaceport has a few basic accommodations, mostly resting areas for the shipyard workers.`

Author: Except for that is no longer in here?

	planet "Canaveral Naval Yard"
		landscape land/space0
		attributes station
		description `To further advance the Republic Navy and secure humanity's future, the government built a new orbital naval yard to allow production of larger and more advanced ships and it remains easily upgradable for any future improvement and expansion. The naval yard is very heavily armed with countless numbers of artillery turrets and advanced anti-missile systems spread across the surface. There's also always a fleet carrier in nearby orbit, always within clear visual range.`

> 		spaceport `The spaceport has a few basic accommodations, mostl resting areas for the shipyard workers.`

Author: ```suggestion
		spaceport `The spaceport has a few basic accommodations, mostly resting areas for the shipyard workers.`
```

> 		bribe .9

Author: ```suggestion
```
